css/style.css

```diff
@@ -131,105 +135,342 @@ li {
 
 /* END Headline Section */
 
-@media screen and (min-width: 768px) {
-  #mobile-design {
-    width: 100%;
-    height: 4561px;
-  }
-
-  header a {
-    color: #fff;
-    transition: all 0.5s ease-out;
-  }
-
-  /* BEGIN Header Menu */
-  .menu {
-    display: none;
-  }
-
-  .name-logo {
-    margin-left: 12.1rem;
-  }
-
-  .navlinks {
-    display: flex;
-    width: 300px;
-    margin-right: 10%;
-    font-family: Inter, sans-serif;
-    font-size: 15px;
-    font-weight: 600;
-    line-height: 20px;
-    align-items: center;
-  }
-
-  .links a {
-    color: #fff;
-    margin-right: 25px;
-    text-decoration-line: none;
-  }
-
-  /* END Header Menu */
-
-  /* BEGIN Headline Section */
-  .headline-desktop {
-    display: flex;
-    flex-direction: row;
-    width: 100%;
-    height: 798px;
-    background: #1c1a19;
-    background-image: url("/images/illustration-header.png");
-    background-repeat: no-repeat;
-    background-position: 100%;
-  }
-
-  .headline-info {
-    width: 566px;
-    height: 739px;
-    display: flex;
-    flex-direction: column;
-    gap: 2rem;
-  }
-
-  .headline-info h2 {
-    margin-left: 20px;
-    font-family: Inter, sans-serif;
-    font-size: 2rem;
-    font-weight: 500;
-    animation-name: text;
-    animation-duration: 3s;
-    animation-iteration-count: infinite;
-  }
-
-  .lateral-bar {
-    width: 15%;
-    display: flex;
-    align-items: center;
-    order: -1;
-  }
-
-  .social-icons {
-    margin-left: 15px;
-    margin-top: 279px;
-    width: 200px;
-    height: 40px;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    gap: 1rem;
-  }
-
-  .support-text-desktop {
-    font-family: Poppins, sans-serif;
-    font-style: normal;
-    font-weight: 400;
-    font-size: 20px;
-    line-height: 32px;
-    color: #fff;
-    margin: 12px 24px 0 20px;
-    gap: 24px;
-    animation-duration: 3s;
-    animation-name: slidein;
-  }
-
-  /* END Headline Section */
+/* BEGIN Works Section */
+.works-section-container {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 3455px;
```

Comment:
- [ Required ] I think it would be better to remove the `height: 3455px` from the styling of the works section this is because it makes the section longer than the length in the Figma design. It is also considered bad practice to give elements on a webpage fixed heights and widths, please rectify this.

## Figma design
<img src="https://user-images.githubusercontent.com/16392046/227380619-cd3fd319-78ab-4c98-818b-6c04a2020cf7.png" width="375px" />


## Your design
<img src="https://user-images.githubusercontent.com/16392046/227380325-6ce688f7-03af-4512-bf4b-4bed76ba3096.png" width="375px" />